helix-term/src/ui/spinner.rs

```diff
@@ -46,22 +55,25 @@ impl Spinner {
     }
 
     pub fn dots(interval: u64) -> Self {
-        Self::new(vec!["⣾", "⣽", "⣻", "⢿", "⡿", "⣟", "⣯", "⣷"], interval)
+        Self::new("⣾⣽⣻⢿⡿⣟⣯⣷", interval)
```

Comment:
Maybe this should be a const on the spinner type so you don't have to duplicate it elsewhere?

Reply:
That's a great idea. I guess it can live in `spinner.rs` and then be imported in the other places it's needed.

Reply:
Add it in the impl for the type.


https://doc.rust-lang.org/reference/items/associated-items.html#associated-constants

Reply:
Made this a constant in `helix-view` so it could be used in `helix-term`.